File: Hahn.ApplicatonProcess.February2021.Data/Asset.cs
using System;

namespace Hahn.ApplicatonProcess.February2021.Data
{
    public class Asset
    {
        public int Id { get; set; }
        public string AssetName { get; set; }
        public string Department { get; set; }
        public string CountryOfDepartment { get; set; }
        public string EMailAdressOfDepartment { get; set; }
        public DateTime PurchaseDate { get; set; }
        public bool? IsBroken { get; set; }
        public bool IsDeleted { get; set; }
    }
}

File: Hahn.ApplicatonProcess.February2021.Web/ClientApp/app/components/asset/asset.ts
import { HttpClient } from 'aurelia-fetch-client';
import { inject } from 'aurelia-framework';

@inject(HttpClient)
export class Asset {
    public assets: AssetModel[] | undefined;

    constructor(http: HttpClient) {
        http.fetch('api/Asset')
            .then(result => result.json())
            .then(data => {
                debugger;
                this.assets = data;
            })
            .catch(error => {
                debugger;
                let tokenResult = {
                    success: false,
                    message: error.message
                };
                return tokenResult;
            });
    }
}

interface AssetModel {
    id: number;
    assetName: string;
    department: string;
    countryOfDepartment: string;
    eMailAdressOfDepartment: string;
    purchaseDate: Date;
    isBroken: boolean;
}